src/cljs/rems/application.cljs

```diff
@@ -758,24 +758,30 @@
                    :on-click #(rf/dispatch [::judge-application "review" (text :t.actions/review)])}])
 
 (defn- third-party-review-button []
-  [button-wrapper {:id "request-review"
+  [button-wrapper {:id "third-party-review"
```

Comment:
`do-third-party-review`?

Reply:
I would think about what logical convention we want to have for naming buttons since that affects UI tests especially.

Reply:
I've now removed the "do-" prefixes, since they're not anymore needed to avoid ID conflicts.